### PR TITLE
improve smb-mount

### LINKED ISSUE #1208: smb-share script - please remove the 3 mount points limit
The smb-share script currently only allows the user to add 3 mount points.
For my current setup I sadly need 7.
It would be great if this limitation could be either removed or increased. :)

**Side-note:**
mount points created by the script are currently the only way how I can get smb shares to work inside nextcloud.
If I add the same SMB shares as external storage inside the GUI of nextcloud then the user is not able to access them.
https://help.nextcloud.com/t/internal-server-error-when-trying-to-access-to-a-smb-share/79678

So increasing the limit from 3 to some higher value would help me out a lot. :)

### PR DESCRIPTION
**Pull request by szaimen**
fix #1208

### CODE REVIEW

**Comment by szaimen** [comment]
@enoch85 Everything is working as expected, hence marking as ready for review.

**Comment by szaimen** [comment]
Please wait a few minutes

**Comment by szaimen** [comment]
it's ready to go now. I finally fixed the external filesystem check.

File: apps/smbmount.sh [hunk 1]
```diff
@@ -240,32 +289,37 @@ fi
...
+            if mountpoint -q $SMBSHARES/$count
+            then
+                msg_box "It seems like the unmount of $SMBSHARES/$count wasn't successful during the deletion. Please try again."
+            else
+                sed -i "/$SMBSHARES_SED\/$count /d" /etc/fstab
+                if grep -q "$SMBSHARES/$count " /etc/fstab
+                then
+                    msg_box "Something went wrong during deletion of $SMBSHARES/$count. Please try again."
+                else
+                    msg_box "Your deletion of $SMBSHARES/$count was successfull!"
```
Comment (enoch85) [inline apps/smbmount.sh:320]: Small typo: should be "successful"

**Comment by enoch85** [comment]
Nice work! Please merge when the typo is fixed.

**Comment by enoch85** [comment]
Please remove the commits (deleting * in the merge request) once you squash and merge please.

File: apps/smbmount.sh (hunk 1 above)
Comment (szaimen) [reply apps/smbmount.sh]: done

**Comment by szaimen** [comment]
> Please remove the commits (deleting * in the merge request) once you squash and merge please.

How do I do that?

**Comment by enoch85** [comment]
Aah you can't merge?

This is how you do it: 
Screenshot from 2020-05-04 11:06:40

Remove all this lines :point_up:

**Comment by szaimen** [comment]
No, I can merge but never have and didn't know thtat you get an option to squash before merging by deleting those lines.

**Pull request merged**